web/modules/custom/bnf/bnf_server/bnf_server.module

```diff
@@ -67,13 +98,12 @@ function bnf_server_theme(): array {
     'bnf_server_import_link' => [
       'variables' => [
         'url' => NULL,
-        'name' => NULL,
         'label' => NULL,
       ],
     ],
     'bnf_server_missing_callback' => [
       'variables' => [
-        'uuid' => NULL,
+        'node_uuid' => NULL,
```

Comment:
Good call. We should fix up `uuid` to `node_uuid`/'subscription_uuid`/`term_uuid` where appropriate, to avoid messing up, in general, but we can fix that up as we go.

Reply:
Yeah, it has also confused me somewhat, that we use `uuid` for everything :)

Reply:
I've shot ourselves in the foot with `subscription_uuid`. It's not the subscriptions uuid, but the subscribed (ie term) uuid.

Reply:
Should we nip it in the bud and fix it now? Before real content is actually added?

Reply:
Well, we already have deployed versions, so we're past the "just rename it" phase and at the "remove the old field, add the new", though we can argue that we can still get away with that without a "migrate data" step as there's no real subscription users yet.

Reply:
I guess it's annoying, but not critical. We'll leave it for now. Agreed?

Reply:
Yeah.